fire-hose-management-frontend/src/components/Departments/DepartmentOverlay.jsx

```diff
@@ -1,158 +1,207 @@
 import * as i18nIsoCountries from "i18n-iso-countries";
-import React, { useState } from "react";
+import React, {useState} from "react";
 import Stack from "@mui/material/Stack";
-import { Button, Grid, MenuItem } from "@material-ui/core";
-import { CTextField } from "../shared/input/CTextField";
-import { CSelect } from "../shared/input/CSelect";
-import { ValidatorForm } from "react-material-ui-form-validator";
-import { Overlay } from "../shared/Overlay";
-import { makeStyles } from "@material-ui/core/styles";
+import {Button, Checkbox, FormControlLabel, Grid, MenuItem} from "@material-ui/core";
+import {CTextField} from "../shared/input/CTextField";
+import {CSelect} from "../shared/input/CSelect";
+import {ValidatorForm} from "react-material-ui-form-validator";
+import {Overlay} from "../shared/Overlay";
+import {makeStyles} from "@material-ui/core/styles";
 
-const useStyles = makeStyles( ( theme ) => ({
-  checkbox: {
-    display: "table-cell",
-    verticalAlign: "middle"
-  },
-  footer: {
-    paddingTop: "24px",
-    paddingBottom: "32px"
-  }
-}) );
+const useStyles = makeStyles((theme) => ({
+    checkbox: {
+        display: "table-cell",
+        verticalAlign: "middle"
+    },
+    footer: {
+        paddingTop: "24px",
```

Comment:
Codacy has a fix for the issue: [Delete `····`](https://app.codacy.com/gh/StitZle/fire-hose-complete/pullRequest?prid=9542260)
```suggestion
    paddingTop: "24px",
```